src/index.html

```diff
@@ -5,9 +5,62 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="../src/styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+
+      <img
+        src="../src/images/imac.jpeg"
+        alt="image"
+        class="card__image"
+      >
+
+      <h1 class="card__title">
```

Comment:
```suggestion
      <h2 class="card__title">
```
Usually, there are a lot of cards on the page, so it's better to use h2-6 here. h1 should be single for the entire page